annif/corpus/subject.py

```diff
@@ -49,6 +49,16 @@ def __len__(self):
     def __getitem__(self, subject_id):
         return (self._uris[subject_id], self._labels[subject_id])
 
+    def append(self, uri, label):
+        subject_id = len(self._uris)
+        self._uris.append(uri)
+        self._labels.append(label)
+        self._uri_idx[uri] = subject_id
+        self._label_idx[label] = subject_id
```

Comment:
This method duplicates much of the code in `__init__(...)`, so if this is kept, maybe there should be a helper method `_append(self, uri, label)` that is used in both? (Also, #385 adds `notation` along `uri` and `label` here.)

Reply:
Sounds like a reasonable refactoring to extract the common code into a method.